### DISCUSSION TITLE
Can't open file in vertical split from picker

### DISCUSSION THREAD

**Discussion by BaronBeans** [helix-editor/helix#8079] [Troubleshooting]
With file picker open the shortcut to open in vertical split is Ctrl-v but this just pastes system clipboard into the file search and there is no way to remap this shortcut.

Ctrl-s to open Horizontal split works fine, but Ctrl-v needs to be remapped.

As described here: https://docs.helix-editor.com/keymap.html#picker
And hard-coded here: https://github.com/helix-editor/helix/blob/fe11ae221812c2aaa917359377f92c511a861886/helix-term/src/ui/picker.rs#L558

**Comment by samnystrom** [comment]
Could you provide more information about your system and terminal emulator? I can't replicate this.

**Comment by philipgiuliani** [comment]
CTRL+v also works fine for me (using macOS), I'm using it all the time.

**Comment by archseer** [comment]
Please fill out the issue template. We don't know your OS, terminal emulator or helix version. This is probably a configuration issue with the terminal emulator.

**Comment by QiBaobin** [comment]
If on Windows, it might be conflict with system shortcut for pasting?

**Comment by vkcku** [comment]
> If on Windows, it might be conflict with system shortcut for pasting?

I'm facing this issue as well and I think this might be the reason.

OS: WSL2 (Ubuntu)
Terminal: Windows Terminal
Shell: zsh

Please let me know if you need any more details.

**Comment by garinger** [comment]
Having the same issue.

OS: Windows 10
Terminal: Windows Terminal
Shell: cmd

**Reply by CptPotato** [reply]
This is not an issue with helix, but Windows Terminal. It intercepts `ctrl-v` for pasting, so it is never actually sent to the editor (see also #10295).
